### PR TITLE
changes for netbsd

### PR DESCRIPTION
**Pull request by Eli-the-Bearded**
NetBSD is a free and open source Unix-like system. It builds and plays with very minimal changes. I think you should include it as a platform.

### CODE REVIEW

File: src/curses.h [hunk 1]
```diff
@@ -15,6 +15,10 @@
   #undef KEY_EVENT
   #include <ncurses/ncurses.h>
 #else
-  #include <ncurses.h>
+  #ifdef __NetBSD__
```
Comment (mrcook) [inline src/curses.h:18]: Is it possible to flatten this indentation?

```
#elif __NetBSD__
```

**Comment by mrcook** [comment]
Many thanks for your PR Eli-the-Bearded, I made just one comment.

File: src/curses.h (hunk 1 above)
Comment (Eli-the-Bearded) [reply src/curses.h]: Done.

**Comment by mrcook** [comment]
Thanks again for the PR Eli, it's much appreciated.

**Pull request merged**